install.sh

```diff
@@ -3819,6 +3820,16 @@ setup_hook_templates() {
         return 0
     fi
 
+    if [ "$(git config --global githooks.maintainOnlyServerHooks)" = "Y" ]; then
+        INSTALL_ONLY_SERVER_HOOKS="yes"
+    fi
+
+    if [ "$INSTALL_ONLY_SERVER_HOOKS" = "yes" ]; then
+        HOOK_NAMES="$MANAGED_SERVER_HOOK_NAMES"
+    else
+        HOOK_NAMES="$MANAGED_HOOK_NAMES"
+    fi
+
     for HOOK in $MANAGED_HOOK_NAMES; do
```

Comment:
Did you need to use `$HOOK_NAMES` perhaps here?

Reply:
Ups.

Reply:
Did you need to change this here?
If not, why the `HOOK_NAMES=...` above?

Reply:
Typo. Thx

Reply:
Hm.. why did the test pass ;_)